src/algebra/group_with_zero/basic.lean

```diff
@@ -291,8 +291,14 @@ instance cancel_monoid_with_zero.to_no_zero_divisors : no_zero_divisors M₀ :=
 ⟨λ a b ab0, by { by_cases a = 0, { left, exact h }, right,
   apply cancel_monoid_with_zero.mul_left_cancel_of_ne_zero h, rw [ab0, mul_zero], }⟩
 
+lemma mul_left_injective' (hc : c ≠ 0) : injective (λ x, x * c) :=
+λ a b, mul_right_cancel' hc
+
```

Comment:
Don't we already have these?

Reply:
https://github.com/leanprover-community/mathlib/blob/e221dc9a044343d75da4f26a8bbb279f3f2df4e7/src/algebra/group_with_zero/defs.lean#L77-L81

Reply:
I failed to find an `injective` version.

Reply:
That's because they were added in a commit that came after you started this PR, it seems.